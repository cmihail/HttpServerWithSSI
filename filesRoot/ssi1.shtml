<html>
<head>
	<title>SSI 1</title>
</head>

<body>
	<!--#include file="included.html" -->
	Random text
	<!--#config timefmt="%A" time="%B" --> <!--#echo   var="DATE_LOCAL" -->
	<!--#include file="included.html" b="other" -->
</body>
</html>
