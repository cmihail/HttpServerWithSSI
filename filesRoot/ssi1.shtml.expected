<html>
<head>
	<title>SSI 1</title>
</head>

<body>
	<b><i>Test</b></i>
<br />
This is a test ....
	Random text
	 
	<b><i>Test</b></i>
<br />
This is a test ....
</body>
</html>
